packages/components/src/components.d.ts

```diff
@@ -214,6 +214,11 @@ export namespace Components {
          */
         "variant": CounterVariant;
     }
+    /**
+     * es-counter demo.
+     */
+    interface EsCounterDemo {
```

Comment:
You need to do a normal build after a "dev" build, for the CI to pass, otherwise you get all these demo types.